openedx/core/djangoapps/theming/helpers.py

```diff
@@ -25,24 +26,28 @@ def get_page_title_breadcrumbs(*args):
 
 def get_value(val_name, default=None, **kwargs):
     """
-    This is a proxy function to hide microsite_configuration behind comprehensive theming.
+    This is a proxy function to hide microsite_configuration and site_configuration behind comprehensive theming.
     """
 
-    # Retrieve the requested field/value from the microsite configuration
-    microsite_value = microsite.get_value(val_name, default=default, **kwargs)
+    if configuration_helpers.is_site_configuration_enabled():
```

Comment:
I know that I suggested in early discussion that we route all site/microsite configuration access (ie, `get_value()` ) through `theming.helpers`, but now that I see it in the code, I am wondering if we should instead route site/microsite configuration access through `site_configuration.helpers`.  It would require updating the various `theming.helpers` access points that current exist in the code.  But since we are already going to be updating the direct microsite_configuration access points, maybe it makes sense to do this now?